services/functional-tests/src/test/java/org/infinispan/online/service/caching/CustomCachingServiceTest.java

```diff
@@ -61,4 +61,11 @@ public void should_have_3_num_owners() {
       assertEquals(3, CommandLine.numOwners(podName));
    }
 
+   @RunAsClient
+   @Test
+   public void should_have_reject_eviction_policy() {
+      String podName = SERVICE_NAME + "-0";
+      assertEquals("EXCEPTION", CommandLine.evictionStrategy(podName));
+   }
+
```

Comment:
I think we should have a test where we ensure an exception is thrown to the client when the capacity is reached.